leaks.go

```diff
@@ -25,14 +26,25 @@ func start(opts *Options) {
 	c := make(chan os.Signal, 2)
 	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
 
+	//get current working directory
+	dir,err := os.Getwd()
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	repoName := getLocalRepoName(opts.RepoURL)
 	fmt.Printf("Cloning \x1b[37;1m%s\x1b[0m...\n", opts.RepoURL)
-	err := exec.Command("git", "clone", opts.RepoURL).Run()
+	r,err := git.PlainClone(dir + "/" + repoName, false, &git.CloneOptions{
+		URL:               opts.RepoURL,
+		RecurseSubmodules: git.DefaultSubmoduleRecursionDepth,
+	})
+	log.Printf("Cloned the repo::",r)
 	if err != nil {
 		log.Printf("failed to clone repo %v", err)
```

Comment:
before the "failed to clone repo" message, call `cleanup` to remove the directory. I was doing some testing and noticed even if go-git fails to clone the directory will still be made